packages/website/.github/pull_request_template.md

```diff
@@ -0,0 +1,19 @@
+---
+### Mandatory Checklist (Delete this from actual PR once completed)
```

Comment:
A left-over from a before time? This should be removed from this PR, as doesn't make much sense in the monorepo.